src/elm/Profile/Contact.elm

```diff
@@ -984,7 +987,7 @@ viewFlagsSelect { t } basic =
             [ flag False "" basic.supportedCountry
             , if basic.showFlags then
                 div
-                    [ class "absolute form-input -mx-px inset-x-0 top-0 space-y-4 z-50 h-44 overflow-auto"
+                    [ class "absolute input -mx-px inset-x-0 top-0 space-y-4 z-50 h-44 overflow-auto"
```

Comment:
There is this weird bug happening....

Its kinda complicate to replicate, maybe there is a simpler route but here we go:

1. edit your contact options
2. Make sure you have something filled for WhatsApp, if not add one and reload the page for a fresh state
2. delete one contact, say telegram
3. fill telegram again back
4. Submit the page
5. go into edit mode again
6. notice how the WhatsApp number is no gone

Here is a demo:
https://user-images.githubusercontent.com/1082127/150033530-811055b5-ccf5-4933-b2a6-44a6dfe1abe8.mov

Reply:
Ok, that was a wild ride! Apparently, it was because we were removing the mask of the phone inputs before validating, but now the mask is automatically removed. So removing the mask was causing the input to be empty, which is the same as deleting it. Should be fixed now!

Reply:
nice bro!